pom.xml

```diff
@@ -48,7 +48,7 @@ under the License.
     </modules>
 
     <properties>
-        <flink.version>1.16.0</flink.version>
+        <flink.version>1.19.0</flink.version>
```

Comment:
I see that we have release [3.0.1-1.17 & 1.16](https://mvnrepository.com/artifact/org.apache.flink/flink-connector-rabbitmq/3.0.1-1.17) in maven repo, This PR will bump the version to 1.19. What is our release plan for 1.18 and will we skip this version